### PR TITLE
fix: SfHeading should be wrapped in div

### LINKED ISSUE #1256: [BUG] SfHeading should be wrapped in div tag
**Describe the bug**
Heading should be wrapped in div tag in order to have it in SfHeader or SfFooter.

**How to reproduce**
1. Place SfHeading in SfHeader
2. See error

**Expected behavior**
No a11y errors 

**Actual behavior**
A11y errors

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue
Closes #1256 

# Scope of work
 SfHeading has been wrapped in div and subtitle prop has been changed on description 

# Visual changes
Migration guide
Zrzut ekranu z 2020-06-29 18-02-28

### CODE REVIEW

File: packages/vue/docs/migration-guide.md [hunk 1]
```diff
@@ -152,5 +152,31 @@ Component | Before | v0.8.0 | Info
...
 
+### Deletions
+
+### Additions
+
+### Changes
+
+- **Props**
+
+Component | Before | v0.8.0
```
Comment (AdamPawlinski) [inline packages/vue/docs/migration-guide.md:163]: Should be changed to v0.9.0

File: packages/vue/docs/migration-guide.md [hunk 2]
```diff
@@ -152,5 +152,31 @@ Component | Before | v0.8.0 | Info
...
+------------ | ------------ | ------------
+SfHeading | subtitle | description
+
+- **Custom properties**
+
+Component | Before | v0.8.0
```
Comment (AdamPawlinski) [inline packages/vue/docs/migration-guide.md:169]: Should be changed to v0.9.0

File: packages/vue/docs/migration-guide.md [hunk 3]
```diff
@@ -152,5 +152,31 @@ Component | Before | v0.8.0 | Info
...
+------------ | ------------ | ------------
+SfHeading | --heading-subtitle-margin | --heading-description-margin
+SfHeading | --heading-subtitle-color | --heading-description-color
+SfHeading | --heading-subtitle-font-size | --heading-description-font-size
+SfHeading | --heading-subtitle-font-family | --heading-description-font-family
+
+- **Slots**
+
+Component | Before | v0.8.0
```
Comment (AdamPawlinski) [inline packages/vue/docs/migration-guide.md:178]: Should be changed to v0.9.0

**Review by AdamPawlinski — COMMENTED** [review]
Just small fix in docs. Everything is all right in my opinion :)

**Review by filrak — APPROVED** [review]

**Pull request merged**